ios/FluentUI/Vnext/List/List.swift

```diff
@@ -151,6 +159,17 @@ class MSFListSectionStateImpl: NSObject, ObservableObject, Identifiable, Control
         }
         let cell = MSFListCellStateImpl()
         cells.insert(cell, at: index)
+        cell.selectionAction = { [weak self] (selectedCell) -> Bool in
+            guard let strongSelf = self, strongSelf.allowsSelection else {
```

Comment:
I added it because I wasn't sure if there were times we'd want one section to be selectable but not another. Like, with the current list demo we have, I don't think it really makes sense to have the persona section be selectable. But, I can simplify it and have it set at just the list level.

Reply:
Do we have any details on the user scenarios we're trying to unlock with this feature? That would help us determine what to support here.